Scripts/fastlane/Fastfile

```diff
@@ -192,19 +192,24 @@ import "./ScreenshotFastfile"
   # This lane builds the app and upload it for both internal and external distribution 
   # -----------------------------------------------------------------------------------
   # Usage:
-  # bundle exec fastlane build_and_upload_release [skip_confirm:<skip confirm>]
+  # bundle exec fastlane build_and_upload_release [skip_confirm:<skip confirm>] [run_on_ci:<run on ci>] 
   #
   # Example:
   # bundle exec fastlane build_and_upload_release 
   # bundle exec fastlane build_and_upload_release skip_confirm:true 
+  # bundle exec fastlane build_and_upload_release run_on_ci:true
   #####################################################################################
   desc "Builds and updates for distribution"
   lane :build_and_upload_release do | options |
+    build_internal = (options[:run_on_ci] == true) ? !ios_validate_ci_build() : true
```

Comment:
Same note here – could we use https://docs.fastlane.tools/actions/is_ci/ instead of an option?

Reply:
Yep! Good catch!
I updated it.